### PR TITLE
moveaxis extension

### PR DESCRIPTION
**Pull request by nassimberrada**

### CODE REVIEW

File: ivy/array/extensions.py [hunk 1]
```diff
@@ -230,3 +230,46 @@ def max_pool2d(
...
+
+        Examples
+        --------
+        >>> x = ivy.zeros((3, 4, 5))
+        >>> print(x.moveaxis(0, -1).shape)
+        (4, 5, 3)
+        >>> print(x.moveaxis(-1, 0).shape)
+        (5, 3, 4)
+        """
+        return ivy.flatten(
```
Comment (Ishticode) [inline ivy/array/extensions.py:271]: This is supposed to be `ivy.moveaxis`

File: ivy/container/extensions.py [hunk 2]
```diff
@@ -493,3 +493,109 @@ def max_pool2d(
...
+        -------
+        ret
+            Container including arrays with moved axes.
+
+        Examples
+        --------
+        With one :class:`ivy.Container` input:
+
+        >>> x = ivy.Container(a=ivy.zeros((3, 4, 5)), b=ivy.zeros((2,7,6)))
+        >>> ivy.static_moveaxis(x, 0, -1).shape
```
Comment (Ishticode) [inline ivy/container/extensions.py:537]: `ivy.Container.static_moveaxis` here

File: ivy/container/extensions.py [hunk 3]
```diff
@@ -493,3 +493,109 @@ def max_pool2d(
...
+        -------
+        ret
+            Container including arrays with moved axes.
+
+        Examples
+        --------
+        With one :class:`ivy.Container` input:
+
+        >>> x = ivy.Container(a=ivy.zeros((3, 4, 5)), b=ivy.zeros((2,7,6)))
+        >>> ivy.moveaxis(x, 0, -1).shape
```
Comment (Ishticode) [inline ivy/container/extensions.py:590]: x.moveaxis(0,-1).shape here

**Review by Ishticode — CHANGES_REQUESTED** [review]
Minor changes required but ready to merge otherwise. Is an issue open for this that I can perhaps link it to?

**Comment by nassimberrada** [comment]
Changes made.
Here's the issue https://github.com/unifyai/ivy/issues/5614

**Review by Ishticode — APPROVED** [review]
Looking good.

**Pull request merged**